test/helpers.js

```diff
@@ -0,0 +1,7 @@
+function simulateChangeEvent(element) {
+  if ('createEvent' in document) {
+    var evt = document.createEvent('HTMLEvents');
+    evt.initEvent('change', false, true);
```

Comment:
`initEvent` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent).

Pls use something like:

```js
var evt = new Event(`change', {bubbles: true, composed: true});
```